### PR TITLE
INSTALL Fixes

### PR DESCRIPTION
**Pull request by jwrober**
For #297

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -112,40 +113,41 @@ Once the build files have been written, then compile with this command:
...
+
+   sudo apt install git \
      cmake \
      ninja-build \
      python3 \
-     gettext \
      qt5-default \
      libkf5archive-dev \
-     libsqlite3-dev \
-     libsdl2-mixer-dev \
```
Comment (lmoureaux) [inline INSTALL:146]: `sdl2-mixer` is needed (but optional) for sound support.

Comment (jwrober) [reply INSTALL:146]: That is why I add it to the list of deps to install after the base `build-dep freeciv` step.

Comment (lmoureaux) [reply INSTALL:146]: Doesn't your patch remove it?

Comment (jwrober) [reply INSTALL:146]: Good catch. Fixed

**Pull request merged**